pkg/parsing/impl/validator.go

```diff
@@ -60,10 +60,6 @@ func (pp *QueryValidator) ValidateCreateTable(query string, chainID tableland.Ch
 		return nil, &parsing.ErrNoTopLevelCreate{}
 	}
 
-	if ast.Errors[0] != nil {
```

Comment:
No more need to check `ast.Errors`, the need for this additional check was confusing